### PR TITLE
fix: postcss vulnerability, move utils for cookie and session handling to @zitadel/next package

### PR DESCRIPTION
**Pull request by peintnermax**
### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] Vitest unit tests ensure that components produce expected outputs on different inputs.
- [ ] Cypress integration tests ensure that login app pages work as expected. The ZITADEL API is mocked.
- [ ] No debug or dead code
- [ ] My code has no repetitions

### CODE REVIEW

File: packages/zitadel-next/src/utils/session.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { getMostRecentCookieWithLoginname } from "./cookies";
+
+const SESSION_LIFETIME_S = 3000;
+
+const transport = createServerTransport(process.env.ZITADEL_SERVICE_USER_TOKEN!, {
+  baseUrl: process.env.ZITADEL_API_URL!,
+  httpVersion: "2",
+});
+
+const sessionService = createSessionServiceClient(transport);
```
Comment (yordis) [inline packages/zitadel-next/src/utils/session.ts:13]: I would careful with this pattern from inside the library; we would be back to the singleton pattern; multi-transport/tenancy wouldn't be possible

Personally, would hold off before going GA with the libraries

File: packages/zitadel-next/src/utils/session.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+export async function loadMostRecentSession(loginName?: string, organization?: string): Promise<Session | undefined> {
+  const recent = await getMostRecentCookieWithLoginname(loginName, organization);
+  return getMostRecentSession(recent.id, recent.token);
+}
+
+async function getMostRecentSession(sessionId: string, sessionToken: string) {
+  return sessionService.getSession({ sessionId, sessionToken }, {}).then((resp) => resp.session);
```
Comment (yordis) [inline packages/zitadel-next/src/utils/session.ts:21]: Personally, I do not see much value on these wrappers that do not add any defaults or extra business invariants inside.

I would call `sessionService.getSession` until then

File: packages/zitadel-next/src/utils/session.ts (hunk 1 above)
Comment (peintnermax) [reply packages/zitadel-next/src/utils/session.ts]: you are right with this one.. how would you make this configurable? I've changed it for the function to get the whole service for now but am not so sure about that..

File: apps/login/src/app/(login)/passkey/add/page.tsx [hunk 3]
```diff
@@ -13,19 +14,8 @@ export default async function Page({
   const { loginName, promptPasswordless, organization, authRequestId } =
     searchParams;
 
-  const sessionFactors = await loadSession(loginName);
+  const sessionFactors = await loadMostRecentSession(loginName, organization);
```
[the remark this answers is not in the document]
Comment (peintnermax) [reply apps/login/src/app/(login)/passkey/add/page.tsx:17]: good input.. I've changed that 😎👍

**Comment by peintnermax** [comment]
@yordis I'd release the proto/client/node packages once we have v2 of both https://github.com/bufbuild/protobuf-es and https://github.com/connectrpc/connect-es and treat the react and next package still as concept until we know more about structuring them 🤔 I appreciate any input there 😎

File: packages/zitadel-next/src/utils/session.ts (hunk 1 above)
Comment (yordis) [reply packages/zitadel-next/src/utils/session.ts]: For now, do not make it configurable; pass the service as an argument. Eventually, we will see the patterns, and we will know what to do about it.

Keep it simple for now. We can always add another layer of indirection wrapping the situation.

**Comment by yordis** [comment]
> react and next package still as concept until we know more about structuring them 🤔 I appreciate any input there 😎

I wouldn't bother with any of those packages and remove them for now; instead, finish up the Login App as soon as possible (meaning, showcase the use cases) and THEN move the code to the appropriate space.

**Pull request merged**